lib/Dialects/LinalgExt/Transforms/Tiling.cpp

```diff
@@ -32,39 +32,47 @@ SmallVector<Value> tileToSCF(PatternRewriter &rewriter, TilingInterface op,
                              TilingInterface clonedOp, ValueRange tileSizes) {
   // Compute lower and upper bounds of the loop nest.
   SmallVector<Range> ranges = clonedOp.getIterationDomain(rewriter);
-  assert(tileSizes.size() == ranges.size() &&
+  assert(tileSizes.size() <= ranges.size() &&
```

Comment:
Yeah I have something similar to make tiling work upstream with padtensor, feel free to update that one too if you need it first.

Reply:
ok, I might need that depending on how we handle pad-tensors.